internal/subscription/domain/subscription.go

```diff
@@ -12,9 +12,9 @@ type SubscriptionFilters struct {
 }
 
 type Course struct {
-	UUID uuid.UUID `db:"uuid" json:"uuid"`
-	Code string    `db:"code" json:"code"`
-	Name string    `db:"name" json:"name"`
+	UUID *uuid.UUID `db:"uuid" json:"uuid"`
+	Code *string    `db:"code" json:"code"`
```

Comment:
The `*string` without `omitempty`... I explain it in another comment.

Reply:
https://github.com/sumelms/microservice-course/pull/28/commits/f25e64978fa73f9551d72ad49f3e2be53b12179b